patches/0004-Use-crypto-backends.patch

```diff
@@ -250,7 +250,7 @@ index b2d4ad7cb0e7f6..f5b109fc6cc069 100644
 +			} else {
 +				*mode = BuildModeExe
 +			}
- 		case "darwin/arm64", "windows/arm", "windows/arm64": // On these platforms, everything is PIE
```

Comment:
`windows/arm` was removed in https://github.com/golang/go/commit/dc54d7b607488e71776ba0fa3d09398d65daf033